packages/runtime/container-runtime/src/test/garbageCollection.spec.ts

```diff
@@ -86,6 +86,8 @@ describe("Garbage Collection Tests", () => {
     // The default GC data returned by `getGCData` on which GC is run. Update this to update the referenced graph.
     const defaultGCData: IGarbageCollectionData = { gcNodes: {} };
 
+    const defaultSnapshotCacheExpiryMs = 2 * 24 * 60 * 60 * 1000;
```

Comment:
This number is everywhere, I wonder if it's smart to export it from the garbageCollection class.